static/js/recent_topics.js

```diff
@@ -168,6 +170,32 @@ exports.update_topic_unread_count = function (message) {
     exports.inplace_rerender(topic_key);
 }
 
+exports.toggle_star_topic = function (stream_id, topic) {
+    const key = stream_id + ":" + topic;
```

Comment:
Let's add a comment explaining here that we're just starring the last message on the topic even though this is a somewhat weird choice.

Reply:
Done.